obspy/clients/fdsn/mass_downloader/mass_downloader.py

```diff
@@ -55,11 +55,13 @@ class MassDownloader(object):
     implementations.
 
     :param providers: List of FDSN client names or service URLS. Will use
-        all FDSN implementations known to ObsPy except RASPISHAKE if set to
-        None. The order in the list also determines their priority, if data
-        is available at more then one provider it will always be downloaded
-        from the provider that comes first in the list. To include RASPISHAKE,
-        you must set this parameter to
+        all FDSN implementations known to ObsPy except RASPISHAKE (generally
+        worse quality data) and IRISPH5 (active source / nodal experiments that
+        might match a very large amount of data occasionally) if set to None.
+        The order in the list also determines their priority, if data is
+        available at more then one provider it will always be downloaded from
+        the provider that comes first in the list. To include RASPISHAKE and
+        IRISPH5, you must set this parameter to
```

Comment:
I forgot that RASPISHAKE is also already excluded - but definitely a sane default to exclude the raspian and irisph5 one!